app/mapper/CaseRequestMapper.scala

```diff
@@ -105,15 +99,15 @@ class CaseRequestMapper {
     )
   }
 
-  private def buildEoriDetailsFromCurrentUser(eoriNumber: String): RegisteredAddressForEori => EORIDetails = { details: RegisteredAddressForEori =>
+  private def toEoriDetails(details: RegisteredAddressForEori): EORIDetails = {
     EORIDetails(
-      eoriNumber,
-      details.field1, // business name
-      details.field2, // address line 1
-      details.field3, // address line 2 (town)
+      details.eori,
+      details.businessName, // business name
```

Comment:
The comments were useful before, when using field1, field2, etc
Now that you named the fields with the correct names, the comments can go away

Reply:
Removed